FAQ.md

```diff
@@ -85,32 +85,11 @@ Right now this is just a collection of links that potentially may be useful:
 
 ## How do I scale a Phaser 3 game? Is there a ScaleManager in v3?
 
-_Last Updated: 2018-12-07_
```

Comment:
Don't entirely drop the previous writeup; 3.16 comes with scale manager but it's worth leaving pointers to the examples in the event that folks are stuck < 3.16 for some reason.